roman-numerals/roman_numerals.rb

```diff
@@ -0,0 +1,77 @@
+class ::Fixnum
+  def method_1000(number,string)
```

Comment:
give proper name to all such method. like to_roman_numeral. and pass digit like 900, 1000 etc. make common method for it